tests/postman/newman/collection.json

```diff
@@ -13576,6 +13576,267 @@
+              "host": [
+                "{{domain}}{{endpoint_customers_me}}"
+              ]
+            }
+          },
+          "response": []
+        },
+        {
+          "name": "Update customer data",
+          "event": [
+            {
+              "listen": "test",
+              "script": {
+                "id": "898b977f-9f9e-40b8-8201-4352c18c619b",
+                "type": "text/javascript",
+                "exec": [
+                  "pm.test(\"Status code is 200\", function () {",
+                  "    pm.response.to.have.status(200);",
+                  "});"
+                ]
+              }
+            },
+            {
+              "listen": "prerequest",
+              "script": {
+                "id": "878ccafd-8752-4982-8896-5f83b8e7aebb",
+                "type": "text/javascript",
+                "exec": [
+                  ""
+                ]
+              }
+            }
+          ],
+          "request": {
+            "auth": {
+              "type": "bearer",
+              "bearer": [
+                {
+                  "key": "token",
+                  "value": "{{customer-access-token}}",
+                  "type": "string"
+                }
+              ]
+            },
+            "method": "POST",
+            "header": [
+              {
+                "key": "Content-Type",
+                "value": "application/json"
+              }
+            ],
+            "body": {
+              "mode": "raw",
+              "raw": "{\n    \"email\": \"should@not.change\",\n    \"firstname\": \"newFirstName\",\n    \"middlename\": \"newMiddleName\",\n    \"lastname\": \"newLastName\"\n}"
+            },
+            "url": {
+              "raw": "{{domain}}{{endpoint_customers}}/{{customer-id}}",
+              "host": [
+                "{{domain}}{{endpoint_customers}}"
+              ],
+              "path": [
+                "{{customer-id}}"
+              ]
+            }
+          },
+          "response": []
+        },
+        {
+          "name": "Check updated customer data",
+          "event": [
+            {
+              "listen": "test",
+              "script": {
+                "id": "703c3a85-b29b-4f52-82ea-34ab57afcf7d",
+                "type": "text/javascript",
+                "exec": [
+                  "pm.test(\"Successfully loaded customer data\", function () {",
+                  "    pm.response.to.have.status(200);",
+                  "});",
+                  "",
+                  "pm.test(\"Customer data was updated\", function () {",
+                  "    const jsonData = pm.response.json();",
+                  "    pm.expect(jsonData.customer_id).to.eq(pm.environment.get(\"customer-id\"));",
+                  "    pm.expect(jsonData.firstname).to.eq('newFirstName');",
+                  "    pm.expect(jsonData.middlename).to.eq('newMiddleName');",
+                  "    pm.expect(jsonData.lastname).to.eq('newLastName');",
```

Comment:
I was planning on adding a custom attribute in the EE version install script and test against it. Can u get it done?